Template/Tests/{PROJECT}Tests/{PROJECT}Tests.swift

```diff
@@ -27,5 +27,16 @@ import XCTest
 import {PROJECT}
 
 class {PROJECT}Tests: XCTestCase {
-    
+
+}
+
+#if os(Linux)
+extension {PROJECT}Tests {
+    static var allTests : [(String, ({PROJECT}Tests) -> () throws -> Void)] {
+        // FIXME: Should return a list of test case (name, function) tuples
+        return [
+          ("testExample", testExample),
```

Comment:
How about we just create a simple test method that does nothing, so that it compiles out of the box?